integration-test/start_solaris_test.go

```diff
@@ -0,0 +1,229 @@
+package main
+
+import (
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/docker/containerd/api/grpc/types"
+	"github.com/docker/docker/pkg/integration/checker"
+	"github.com/go-check/check"
+	"google.golang.org/grpc"
+)
+
+func (cs *ContainerdSuite) TestStartBusyboxLsSlash(t *check.C) {
+	expectedOutput := `bin
+dev
+etc
+export
+home
+lib
+mnt
+opt
+proc
+root
+sbin
+system
+tmp
+usr
+var
+`
+	if err := CreateSolarisBundle("busybox-ls-slash", []string{"ls", "/"}); err != nil {
```

Comment:
I'd rather not have 2 files to maintain for tests.

Could you rename `CreateBusyboxBundle` to something like `CreateTestBundle` and have it choose between `busybox` or `rootfs` depending on the current OS?